teams/mentorship-programs.toml

```diff
@@ -21,3 +21,7 @@ alumni = []
 
 [[zulip-groups]]
 name = "gsoc-mentors"
+
+[website]
+name = "Mentorship Programs"
+description = "Organizes involvement in internship programs such as Google Summer of Code"
```

Comment:
I'd suggest we deconflict this by first merging:

- https://github.com/rust-lang/team/pull/1671/files